### ISSUE TITLE
Section 508 Audit | Remediate non-conformant issues (25633)

### ISSUE THREAD

**Issue by nikitarockz** [department-of-veterans-affairs/caseflow#5110]
As Project Manager, I would like the section 508 non-conformant issues that were identified in the test performed on 3/26/2018 on this product so that we are in compliant with our ATO requirements.

**_Just the Facts_**

Product | Results | Environment
-- | -- | --
Hearing Prep | Non-compliant | http://dsva-appeals-certification-demo-1715715888.us-gov-west-1.elb.amazonaws.com/test/users

### Finding - [See Audit Notes](https://github.com/department-of-veterans-affairs/caseflow/issues/5110#issuecomment-379881489)

Defect | Defect Description | Functional Performance   Criteria | Severity | Recommended   Resolution | Status | Comments
-- | -- | -- | -- | -- | -- | -- 
1 | Text content is not tagged properly for assistive technology users. | 1.3.1 | Critical | Ensure text content is tagged properly for assistive technology users. | New | See Note 1
2 | The name, role, and    value of elements cannot be read by assistive technology programs. | 4.1.2 | Critical | Make sure the name, role, and value of all elements can be read    by assistive technology programs. | New | See Note 2
3 | A well-defined    on-screen indication of the current focus is not provided. | 2.4.7 | Critical | Ensure a well-defined on-screen focus is provided for all    elements within the web application. | New | See Note 3
4 | Content cannot be    accessed using a keyboard. | 2.1.1 | Critical | Make sure all web elements can be accessed using a keyboard. | New | See Note 4
5 | Content does not    have sufficient color contrast to be distinguished by users with visual    impairments. | 1.4.3 | Medium | Ensure content has sufficient color contrast to be    distinguished by users with visual impairments. | New | See Note 5
6 | Field labels are not read for assistive    technology users. | 3.3.2 | Critical | Make sure field labels can be read by assistive technology    programs. | New | See Note 6

# Acceptance Criteria
1. Verify that a [Remediation Plan](https://github.com/department-of-veterans-affairs/caseflow/issues/5110#issuecomment-381247374) has been provided within 2 weeks.
1. Verify that these non-conformant issues have/has been reconciled, remediation planned, or justification provided.  

# Related Issues
Section 508 | Hearing Prep | Request remediation of non-conformant issues (25633) [#1930](https://github.com/department-of-veterans-affairs/appeals-pm/issues/1930)

**_Engineering_**
Section 508 Audit | Remediate non-conformant issues (25633) #5110

**_PRs_**
https://github.com/department-of-veterans-affairs/caseflow/pull/5752
https://github.com/department-of-veterans-affairs/caseflow/pull/5694

**Comment by lakohl** [comment]
cc @annienguyenUSDS

**Comment by nikitarockz** [comment]
# Remediation Plan
All issues will be addressed within 3 sprints.  Some will be address sooner do to UI design changes.

**Comment by lakohl** [comment]
@annienguyenUSDS  @nikitarockz I believe @nolawi  has finished all the tickets related to these issues. Perhaps we should do an internal double-check of these issues and then have the 508 team re-test?

**Comment by lakohl** [comment]
@annienguyenUSDS when HP demo is stabilized we could ask the 508 team to re-test.

**Comment by lakohl** [comment]
I made this ticket to update the environments page with instructions for accessing our apps: https://github.com/department-of-veterans-affairs/caseflow/issues/6441 
Once it's updated, we'll need to provide that info to the 508 team as well. See https://github.com/department-of-veterans-affairs/appeals-pm/issues/2101

**Comment by nikitarockz** [comment]
# Re-opening
All tasks have not been completed.

**Comment by nicholasholtz** [comment]
@nikitarockz How do we know that all tasks have not been completed? Has this been retested by the VA 508 team? Can you provide more specifics to explain why Nolawi's note of 10/4 is not accurate? Thanks.

**Comment by sharonwarner** [comment]
@nikitarockz @nicholasholtz 

We have definitely fixed *some* of these issues, but not all of them. Is it possible to hold off on the retest until after our February deadline? It will take some time to go through and confirm which issues have been corrected.
